sda-admin/go.mod

```
@@ -1,6 +1,6 @@
 module github.com/neicnordic/sensitive-data-archive/sda-admin
 
-go 1.22.3
+go 1.23
```

```suggestion
go 1.23.1
```

This needs the patch version as well or you will get errors like `toolchain not available`

---

Fixed